services/metadata/frontend/BurgerButton.svelte

```diff
@@ -0,0 +1,43 @@
+<script>
+  export let type = "open";
+</script>
+
+<div class="button" on:click>
+  {#if type=="open"}
```

Comment:
Use strict operator `===`.

Btw, this whole custom-made burger component can be replaced by 2 icons, but for now we can keep it.

Reply:
done